signac/contrib/project.py

```diff
@@ -1827,3 +1857,16 @@ def get_project(root=None):
     :rtype: :py:class:`~.Project`
     :raises LookupError: If no project configuration can be found."""
     return Project.get_project(root=root)
+
+
+def get_job(root=None):
+    """From a job workspace directory, locate the closest parent project
+    and return this job.
+
```

Comment:
Can we maybe add a simple example on how this function is to be used here?

Reply:
Added an example.